modules/apps/document-library/document-library-test/src/testIntegration/java/com/liferay/document/library/service/test/DLFileEntryServiceTest.java

```diff
@@ -217,6 +221,56 @@ public void testCopyFileEntryWithoutExtensionInRootFolderToFolder()
 			serviceContext);
 	}
 
+	@Test
+	public void testGetFileEntries() throws Exception {
+		DLFileEntry dlFileEntry1 = addDLFileEntry(
+			DLFolderConstants.DEFAULT_PARENT_FOLDER_ID, false);
+		DLFileEntry dlFileEntry2 = addDLFileEntry(
+			DLFolderConstants.DEFAULT_PARENT_FOLDER_ID, false);
+		DLFileEntry dlFileEntry3 = addDLFileEntry(
+			DLFolderConstants.DEFAULT_PARENT_FOLDER_ID, false);
+
+		_ratingsEntryLocalService.updateEntry(
+			TestPropsValues.getUserId(), DLFileEntry.class.getName(),
+			dlFileEntry1.getFileEntryId(), 1.0,
+			ServiceContextTestUtil.getServiceContext(_group.getGroupId()));
+
+		_ratingsEntryLocalService.updateEntry(
+			TestPropsValues.getUserId(), DLFileEntry.class.getName(),
+			dlFileEntry2.getFileEntryId(), 1.0,
+			ServiceContextTestUtil.getServiceContext(_group.getGroupId()));
+
+		_ratingsEntryLocalService.updateEntry(
+			TestPropsValues.getUserId(), DLFileEntry.class.getName(),
+			dlFileEntry3.getFileEntryId(), 0.5,
+			ServiceContextTestUtil.getServiceContext(_group.getGroupId()));
+
+		Assert.assertEquals(
+			2,
+			DLFileEntryServiceUtil.getFileEntriesCount(
+				_group.getGroupId(), 1.0));
+		Assert.assertEquals(
+			Arrays.asList(dlFileEntry2, dlFileEntry1),
+			DLFileEntryServiceUtil.getFileEntries(
+				_group.getGroupId(), 1.0, QueryUtil.ALL_POS,
+				QueryUtil.ALL_POS));
+
+		_ratingsEntryLocalService.updateEntry(
+			TestPropsValues.getUserId(), DLFileEntry.class.getName(),
+			dlFileEntry1.getFileEntryId(), 1.0,
+			ServiceContextTestUtil.getServiceContext(_group.getGroupId()));
+
+		Assert.assertEquals(
+			2,
+			DLFileEntryServiceUtil.getFileEntriesCount(
+				_group.getGroupId(), 1.0));
+		Assert.assertEquals(
+			Arrays.asList(dlFileEntry1, dlFileEntry2),
+			DLFileEntryServiceUtil.getFileEntries(
+				_group.getGroupId(), 1.0, QueryUtil.ALL_POS,
+				QueryUtil.ALL_POS));
```

Comment:
We can see that it worked because we're checking the the order of the returned file entries has changed. The count check is there just for completeness.